### PR TITLE
Update 简体中文 (Simplified Chinese).json

### PR DESCRIPTION
**Pull request by SummonHIM**
Optimize translation for Simplified Chinese.
I added some spaces after some descriptions to prevent the font becoming too large.

### CODE REVIEW

File: src/Languages/简体中文 (Simplified Chinese).json [hunk 1]
```diff
@@ -12,55 +12,54 @@
...
+  "back_button": "返回",
   "mount_button": "安装",
-  "mounted": "安装完成！",
+  "mounted": "已安装！",
   "mount_complete": "安装完成！",
-  "mounting_mod": "安装模块！",
+  "mounting_mod": "正在安装！",
   "unmount_button": "卸载",
-  "unmounted_toast": "未装载!",
-  "args-error": "启动参数解析失败！\n仅使用默认启动参数！\n请查看日志获取更多信息！",
```
Comment (OrsellGaming) [inline src/Languages/简体中文 (Simplified Chinese).json:22]: It seems you accidentally removed this translation

**Review by OrsellGaming — CHANGES_REQUESTED** [review]
It seems you accidentally removed this translation

File: src/Languages/简体中文 (Simplified Chinese).json (hunk 1 above)
Comment (OrsellGaming) [reply src/Languages/简体中文 (Simplified Chinese).json:22]: The "args-error" one to be more specific

Comment (SummonHIM) [reply src/Languages/简体中文 (Simplified Chinese).json:22]: I didn't see 'args-error' in english.json, so I removed it.

Comment (OrsellGaming) [reply src/Languages/简体中文 (Simplified Chinese).json:22]: Its right here https://github.com/Portal-2-Multiplayer-Mod/Portal-2-Multiplayer-Mod/blob/693c664954536ec9e5f29c886fa0395aaa75ecc9/src/Languages/English.json#L22

Comment (SummonHIM) [reply src/Languages/简体中文 (Simplified Chinese).json:22]: Oh, my fault. I used the English.json from the main branch to compare. It's fixed now.

**Review by OrsellGaming — APPROVED** [review]

**Pull request merged**